erroranalysis/tests/test_surrogate_error_tree.py

```diff
@@ -41,6 +47,44 @@ def test_surrogate_error_tree_int_categorical(self):
         run_error_analyzer(model, X_test, y_test, list(X_train.columns),
                            categorical_features)
 
+    def test_large_data_surrogate_error_tree(self):
+        # validate tree trains quickly for large data
+        X_train, y_train, X_test, y_test, _ = \
+            create_binary_classification_dataset(100)
+        feature_names = list(X_train.columns)
+        model = create_sklearn_random_forest_regressor(X_train, y_train)
+        for _ in range(16):
+            X_test = pd.concat([X_test, X_test], ignore_index=True)
+            y_test = np.concatenate([y_test, y_test])
+        assert X_test.shape[0] > 1000000
```

Comment:
Can you move the above logic to create a large test set into a utility function in common_utils?

Reply:
done!